main.c

```diff
@@ -1,18 +1,39 @@
 #include <stdlib.h>
 #include <stdio.h>
+
 int main(void)
 {
-	int lenght ;
+	int lenght;
 	printf("input lenght");
-	scanf("%d",&lenght);
+	scanf("%d", &lenght);
 	printf("lenght - %d  \n", lenght);
 	int array[lenght];
 	int *ptr = array;
+
 	for(int i = 0; i < lenght; i++)
 	{
-		printf("input number %d \n",i);
-		scanf("%d",(ptr+i));
+		printf("input number %d \n", i);
+		scanf("%d", (ptr+i));
 		printf("element - %d : %d \n", i, *(ptr+i));
 	}
-	
-}
\ No newline at end of file
+
+	for(int i = 0; i < lenght - 1; i++)
+	{
+		for(int j = 0; j < lenght - i -1; j++)

```

Comment:
It is a good practice to use brackets in those cases (j < lenght - i -1)

Reply:
Done